src/meta/expr.lean

```diff
@@ -326,6 +326,16 @@ meta def nat.to_pexpr : ℕ → pexpr
 | 0 := ``(0)
 | 1 := ``(1)
 | n := if n % 2 = 0 then ``(bit0 %%(nat.to_pexpr (n/2))) else ``(bit1 %%(nat.to_pexpr (n/2)))
+
+/--
+`int.to_pexpr n` creates a `pexpr` that will evaluate to `n`.
+The `pexpr` does not hold any typing information:
```

Comment:
```suggestion
Unlike `int.has_reflect`, the resulting `pexpr` does not hold any typing information:
```

Reply:
I'm not sure that calling the difference out here is necessary, since the types are different (`int.has_reflect` doesn't produce a `pexpr`).